src/ServiceControl.Audit.Persistence.RavenDB/CustomChecks/CheckFreeDiskSpace.cs

```diff
@@ -0,0 +1,84 @@
+﻿namespace ServiceControl.Audit.Persistence.RavenDB.CustomChecks
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using NServiceBus.CustomChecks;
+    using NServiceBus.Logging;
+    using RavenDB;
+
+    class CheckFreeDiskSpace : CustomCheck
+    {
+        public CheckFreeDiskSpace(DatabaseConfiguration databaseConfiguration) : base("ServiceControl.Audit database", "Storage space", TimeSpan.FromMinutes(5))
+        {
+            dataPath = databaseConfiguration.ServerConfiguration.DbPath;
+            percentageThreshold = databaseConfiguration.DataSpaceRemainingThreshold / 100m;
+            Logger.Debug($"Check ServiceControl data drive space remaining custom check starting. Threshold {percentageThreshold:P0}");
+        }
+
+        public override Task<CheckResult> PerformCheck(CancellationToken cancellationToken = default)
+        {
+            var dataPathRoot = Path.GetPathRoot(dataPath);
```

Comment:
`dataPath` itself can still be `null` here, when an external RavenDB instance is being used via the `ConnectionString` property on `ServerConfiguration`\`RavenPersisterSettings`.

All 4 of these custom checks should have logic that skips the logic and passes the checks when an external RavenDB server is being used.

It looks like there is a `UseEmbeddedServer` property available on both config types that should directly serve that purpose vs. having to infer it because of a `null` `DbPath`.

Then, if we're embedded, and `DbPath` is `null` that does seem like a throwing scenario.